lib/ClangImporter/ImportDecl.cpp

```diff
@@ -9369,13 +9404,27 @@ void ClangImporter::Implementation::loadAllMembersOfObjcContainer(
 
 void ClangImporter::Implementation::insertMembersAndAlternates(
     const clang::NamedDecl *nd, SmallVectorImpl<Decl *> &members) {
+  // If this declaration shouldn't be visible, don't add it to
+  // the list.
+  if (shouldSuppressDeclImport(nd))
+    return;
+
+  size_t start = members.size();
   llvm::SmallPtrSet<Decl *, 4> knownAlternateMembers;
+  Decl *asyncImport = nullptr;
   forEachDistinctName(
       nd, [&](ImportedName name, ImportNameVersion nameVersion) -> bool {
     auto member = importDecl(nd, nameVersion);
     if (!member)
       return false;
 
+    members.push_back(member);
+    if (nameVersion.supportsConcurrency()) {
+      assert(!asyncImport &&
```

Comment:
Can this be hit with normal code, or would something more catastrophic need to take place to hit this?
If it can be hit, is there a way to plumb an error message through the `ClangImporter`? I do see some initialization of what looks to be a diagnostic engine, but I'm not sure how accessible it actually is.

Reply:
It would only be hit if forEachDistinctName added a new callback with another version that had supportsConcurrency, so it really should never be hit.